docs/.vitepress/config.ts

```diff
@@ -140,7 +140,7 @@ const config = defineConfig({
         link: '/api/',
       },
       {
-        text: 'Ecosystem',
+        text: 'Try it',
         items: [{ text: 'StackBlitz ', link: 'https://fakerjs.dev/new' }],
```

Comment:
Should we put the link directly in there instead of a nested entry?
```suggestion
        link: 'https://fakerjs.dev/new',
```

Reply:
i dont mind the nesting, it kinda gives you a warning you will be thrown off to a completely different site which might be disconcerting otherwise.